hack/make-rules/prow-images.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
I'm curious why this is under the `hack/make-rules` directory. Did you intend to add Makefile targets to use this script? Since this is specific to Prow not the whole repo, why not define it under `prow`?

Reply:
good point, moved over to be under prow